### PR TITLE
Develop/fixes/dlsv2 687 page 500 error on choose working group members screen when given user email not associated with dls account

### PR DESCRIPTION
**Pull request by OluwatobiAwe**
### JIRA link
[DLSV2-687](https://hee-dls.atlassian.net/browse/DLSV2-687)

### Description
Error not properly displayed to conform with all other error across the system.

### Screenshots
ErrorDisplay (2)

-----
### Developer checks
(Leave tasks unticked if they haven't been appropriate for your ticket.)

I have:
- [x] Run the formatter and made sure there are no IDE errors (see [info on Text Editor settings](https://hee-tis.atlassian.net/wiki/spaces/TP/pages/3546185813/DLS+Dev+Process) to avoid whitespace changes)
- [ ] Written tests for the changes (accessibility tests, unit tests for controller, data services, services, view models, etc)
- [ ] Manually tested my work with and without JavaScript
- [x] Tested any Views or partials created or changed with [Wave Chrome plugin](https://chrome.google.com/webstore/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh/related) and addressed any valid accessibility issues
- [ ] Updated/added documentation in [Confluence](https://hee-tis.atlassian.net/wiki/spaces/TP/pages/3546939432/DLS+Code) and/or [GitHub Readme](https://github.com/TechnologyEnhancedLearning/DLSV2/blob/master/README.md). List of documentation links added/changed:
  - [doc_1_here](link_1_here)
- [x] Updated my Jira ticket with information about other parts of the system that were touched as part of the MR and have to be sanity tested to ensure nothing’s broken
- [x] Scanned over my own MR to ensure everything is as expected and it looks right in the browser

### CODE REVIEW

**Comment by OluwatobiAwe** [comment]
So I decided not to use the view Component specifically created for displaying error such as this because the field doesn't belong to a strongly typed model where error messages and validation messages could be added.

File: DigitalLearningSolutions.Web/Controllers/FrameworksController/Frameworks.cs [hunk 1]
```diff
@@ -680,11 +680,11 @@ public IActionResult AddCollaborator(string actionname, string userEmail, bool c
                 }
                 else if (collaboratorId == -2)
                 {
-                    TempData["FrameworkError"] = $"User with the email address {userEmail} has been previously added";
+                    TempData["FrameworkError"] = $"User with the email address has been previously added";
                 }
                 else if (collaboratorId == -4)
                 {
-                    TempData["FrameworkError"] = $"The email address {userEmail}  must match registered DLS Admin account";
+                    TempData["FrameworkError"] = $"The email address must match registered DLS Admin account";
```
Comment (kevwhitt-hee) [inline DigitalLearningSolutions.Web/Controllers/FrameworksController/Frameworks.cs:687]: Tweak to wording:

"The email address must match registered **a** DLS Admin account"

**Review by kevwhitt-hee — CHANGES_REQUESTED** [review]
See changes requested in code.

File: DigitalLearningSolutions.Web/Views/Frameworks/Developer/Collaborators.cshtml [hunk 2]
```diff
@@ -4,13 +4,29 @@
...
+  string errMsg = TempData["FrameworkError"]?.ToString();
   if (!string.IsNullOrWhiteSpace(errMsg)) { Model.Error = true; }
 }
 <link rel="stylesheet" href="@Url.Content("~/css/frameworks/frameworksShared.css")" asp-append-version="true">
 @section NavMenuItems {
   <partial name="Shared/_NavMenuItems" />
 }
+@if (Model.Error)
+{
+    @*<vc:error-summary order-of-property-names="@(new string[] { "userEmail" })" />*@
```
[the remark this answers is not in the document]
Comment (OluwatobiAwe) [reply DigitalLearningSolutions.Web/Views/Frameworks/Developer/Collaborators.cshtml]: fixed

File: DigitalLearningSolutions.Web/Controllers/FrameworksController/Frameworks.cs (hunk 1 above)
Comment (OluwatobiAwe) [reply DigitalLearningSolutions.Web/Controllers/FrameworksController/Frameworks.cs]: fixed

File: DigitalLearningSolutions.Web/Controllers/FrameworksController/Frameworks.cs [hunk 3]
```diff
@@ -680,11 +680,11 @@ public IActionResult AddCollaborator(string actionname, string userEmail, bool c
                 }
                 else if (collaboratorId == -2)
                 {
-                    TempData["FrameworkError"] = $"User with the email address {userEmail} has been previously added";
+                    TempData["FrameworkError"] = $"User with the email address has been previously added";
```
[the remark this answers is not in the document]
Comment (OluwatobiAwe) [reply DigitalLearningSolutions.Web/Controllers/FrameworksController/Frameworks.cs]: fixed

**Review by kevwhitt-hee — APPROVED** [review]
Looks good. Please avoid committing fixes for different tickets in the same PR in future, though.

**Pull request merged**